helm/keycloak/values.yaml

```diff
@@ -112,26 +142,18 @@ ingress:
     nginx.ingress.kubernetes.io/proxy-buffer-size: "16k"
     nginx.ingress.kubernetes.io/proxy-buffers-number: "4"
 
-containerPorts:
-  - name: web
-    port: 8080
-    protocol: TCP
-  - name: mgmt
-    port: 8090
-    protocol: TCP
-
 service:
   type: ClusterIP
```

Comment:
I don't think `service.type` should be configurable if we need a headless service for pod discovery.

```suggestion
```